client.go

```diff
@@ -176,6 +176,8 @@ type Client interface {
 	Close() error
 }
 
+var _ Client = &ovndb{}
```

Comment:
Why this change?

Reply:
Just to ensure that the `ovndb` struct  implements the methods defined by the `Client` interface